tests/testsuite/collisions.rs

```diff
@@ -55,12 +55,11 @@ fn collision_dylib() {
     p.cargo("build -j=1")
         .with_stderr_data(&format!("\
 ...
-[WARNING] output filename collision.
-The lib target `a` in package `b v1.0.0 ([ROOT]/foo/b)` has the same output filename as the lib target `a` in package `a v1.0.0 ([ROOT]/foo/a)`.
-Colliding filename is: [ROOT]/foo/target/debug/deps/{}a{}
-The targets should have unique names.
-Consider changing their names to be unique or compiling them separately.
-This may become a hard error in the future; see <https://github.com/rust-lang/cargo/issues/6313>.
+[WARNING] output filename collision at [ROOT]/foo/target/debug/deps/{}a{}
```

Comment:
Should we move this into an [`Origin`](https://docs.rs/annotate-snippets/latest/annotate_snippets/struct.Origin.html)?  I *think* not but unsure

CC @Muscraft